extensions/wikia/UserLogin/js/FacebookFormConnectUser.js

```diff
@@ -0,0 +1,29 @@
+/* global UserBaseAjaxForm */
+
+(function () {
+	'use strict';
+
+	var FacebookFormConnectUser = function (el, options) {
+		UserBaseAjaxForm.call(this, el, options);
+	};
+
+	FacebookFormConnectUser.prototype = Object.create(UserBaseAjaxForm.prototype);
```

Comment:
We're good: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create#Browser_compatibility